upcloud/managed_database.go

```diff
@@ -622,10 +622,10 @@ type ManagedDatabaseUserPGAccessControl struct {
 }
 
 type ManagedDatabaseUserRedisAccessControl struct {
```

Comment:
Do we need similar changes for other DB types as well? Those seem to not use `omitempty`, though 🤔

Reply:
Good point, those are changed  now as well.